### ISSUE TITLE
Add term thickened cortex of the mandible

### ISSUE THREAD

**Issue by lisannesw** [obophenotype/human-phenotype-ontology#10455]
Preferred term label:
Thickened cortex of the mandible

Synonyms
Broad cortex of mandible- Cortical thickening of the mandible- Thickened cortices of mandible

Definition (free text, please give PubMed ID)
Abnormal thickening of the cortex of the mandible. PMID24389367

Parent term (use hpo.jax.org/app)
Abnormal mandible morphology HP:0000277

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)
Caffey Disease (omim114000), for example: PMID24390061

Your nano-attribution (ORCID)
Andreas Zankl: 0000-0001-8612-1062, @azankl
Lisanne Wit: 0009-0007-1020-5412

**Comment by pnrobinson** [comment]
@azankl

Adding new term: 
- ID: HP:6000937
- label: Thickened cortex of the mandible
- parents: [Abnormal mandible morphology (HP:0000277)](https://hpo.jax.org/app/browse/term/HP:0000277)
- pmid: PMID:24389367|PMID:24390061
- def: Abnormal thickening of the cortex of the mandible.
- comment: 
- orcid: orcid:0000-0001-8612-1062
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10455
- synonyms: Thickened cortices of mandible(NONE)|Cortical thickening of the mandible(NONE)|Broad cortex of mandible(NONE)
